src/test/01-manipulating-fabrics.ts

```diff
@@ -202,11 +204,15 @@ describe("Manipulating fabrics", function () {
             await fabric.collection(name).document("example");
             expect.fail("Expected document to be destroyed");
           } catch (e) {
-            expect(e).to.be.an.instanceof(C8Error);
-            expect(e.code).eq(404);
+            let message;
+            if (e instanceof C8Error) {
+              expect(e).to.be.an.instanceof(C8Error);
```

Comment:
message will be undefined if it goes in e and that will be consoled , You can remove else condition if you move return inside of if and then you can use const message= String(e) outside if

Reply:
👍